account_payment_partner/security/account_payment_partner_security.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="group_show_account_payment_method" model="res.groups">
+        <field name="name">Show the default payment method in contacts</field>
```

Comment:
We should specify something else so that this group does not cause confusion when viewing the option from the user's permissions.
If I read it as it is, I'm sure I'll mark it. :wink: 
```suggestion
        <field name="name">Show the default payment method line in contacts</field>
```